cli/flox-rust-sdk/src/models/manifest/raw.rs

```diff
@@ -228,8 +302,31 @@ impl RawManifest {
                 # myservice.command = "python3 -m http.server""#});
 
         manifest.insert(MANIFEST_SERVICES_KEY, Item::Table(services_table));
+    }
+
+    /// Populates an example include section.
+    fn add_include_section(manifest: &mut DocumentMut) {
+        let mut include_table = Table::new();
+
+        include_table.decor_mut().set_prefix(indoc! {r#"
+
+                
+                 ## Include ----------------------------------------------------------
+                 ## ... include other environments to merge with this one
```

Comment:
Since you asked for input.. the wording sounded like the order was reversed.
```suggestion
                 ## ... include other environments for this one to be merged onto
```

Reply:
> ... include other environments to merge with this one

This wording makes sense to me.

> ... include other environments for this one to be merged onto

If this is more correct, I think the tradeoff is that it's not as clear.

I have a few proposals, with increasing detail, but I like the first, shortest one best: 

```suggestion
                 ## ... environments used to create a merged manifest
```

```suggestion
                 ## ... dependent environments used to create a merged manifest
```

```suggestion
                 ## ... environments that will be merged with this environment
                 to create a merged manifest
```

Reply:
> include other environments for this one to be merged onto

This actually sounds backwards to me lol

My nitpick with `to create a merged manifest` is that this comment is for someone that doesn't know this feature exists already, and in that situation I think merging environments gives a clearer picture of what this feature does than merging manifests.

Reply:
Yeah, I guess that either variation could be read in either direction. Maybe I've just been staring at the implementation too long. I don't feel strongly about changing it from what the PR has.

Reply:
We looked at all the wording used for composition and landed on a new description for this section:

```
Include ----------------------------------------------------------
... environments to create a composed environment
------------------------------------------------------------------
```

Reply:
Done